auto/update_draft_picks.R

```diff
@@ -1,43 +1,68 @@
 library(dplyr)
-library(tibble)
-library(purrr)
-library(tidyr)
-library(rvest)
-library(stringr)
-library(janitor)
-library(glue)
-library(nflreadr)
+# library(tibble)
+# library(purrr)
+# library(tidyr)
+# library(rvest)
+# library(stringr)
+# library(janitor)
+# library(glue)
+# library(nflreadr)
 
 
 scrape_draft <- function(year = nflreadr::most_recent_season(roster =  TRUE)) {
 
+  Sys.sleep(3)
+
   html_scrape <- rvest::read_html(glue::glue("https://www.pro-football-reference.com/years/{year}/draft.htm"))
 
-  table_node <- html_scrape %>%
+  table_node <- html_scrape |>
     rvest::html_element("#drafts")
 
-  cfb_ids <- rvest::html_elements(table_node, "tr th[data-stat='college_link'],td[data-stat='college_link']") %>%
-    rvest::html_element("a") %>%
-    rvest::html_attr("href") %>%
-    stringr::str_remove_all(".+sports-reference.com/cfb/players/|.html") %>%
+  cfb_ids <- rvest::html_elements(table_node, "tr th[data-stat='college_link'],td[data-stat='college_link']") |>
+    rvest::html_element("a") |>
+    rvest::html_attr("href") |>
+    stringr::str_remove_all(".+sports-reference.com/cfb/players/|.html") |>
     tail(-1)
 
-  pfr_ids <- table_node %>%
-    rvest::html_elements("tr th[data-stat='player'],td[data-stat='player']") %>%
-    html_attr("data-append-csv") %>%
+  pfr_ids <- table_node |>
+    rvest::html_elements("tr th[data-stat='player'],td[data-stat='player']") |>
+    rvest::html_attr("data-append-csv") |>
     tail(-1)
 
-  draft_table <- table_node %>%
+  roster <- tibble::tibble(gsis_id = character(),
+                           pfr_id = character())
+
+  try({
+    roster <- nflreadr::load_rosters(seasons = TRUE) |>
+      dplyr::filter(!is.na(pfr_id),!is.na(gsis_id)) %>%
+      dplyr::distinct(gsis_id, pfr_id)
+  },silent = TRUE)
+
+  draft_table <- table_node |>
     rvest::html_table() %>%
-    {suppressWarnings(janitor::row_to_names(.,row_number = 1))} %>%
+    {suppressWarnings(janitor::row_to_names(.,row_number = 1))} |>
```

Comment:
Base pipe needs underscore as placeholder here imo

Reply:
uses magrittr pipe because I don't have r4.2.0 on my dev yet and thus cannot test for a while